tax_form
2016-04-03 03:29:01.773
ruby -v -W0 /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb
ruby 2.2.4p230 (2015-12-16) [x86_64-linux-gnu]
Run options: --seed 0

# Running:

.......ruby script/workflow.rb --test README
confirm_commit(:echo @repository.something_to_commit?=true
.E........confirm_commit(:echo @repository.something_to_commit?=false
..confirm_commit(:echo @repository.something_to_commit?=true
..E...confirm_commit(:echo @repository.something_to_commit?=true
ruby script/workflow.rb --test README
confirm_commit(:echo @repository.something_to_commit?=true
ruby script/workflow.rb --test README
confirm_commit(:echo @repository.something_to_commit?=true
ruby script/workflow.rb --test README
confirm_commit(:echo @repository.something_to_commit?=true
ruby script/workflow.rb --test README
confirm_commit(:echo @repository.something_to_commit?=true
.$ git rebase --abort
@errors="No rebase in progress?\n"
@process_status=#<Process::Status: pid 25266 exit 1>
$ git merge --abort
@errors="fatal: There is no merge to abort (MERGE_HEAD missing).\n"
@process_status=#<Process::Status: pid 25287 exit 128>
$ git checkout master
@errors="Already on 'master'\n"
....

Finished in 139.418453s, 0.2152 runs/s, 1.4632 assertions/s.

  1) Error:
NewTestClass#test_calc_test_maturity:
ArgumentError: wrong number of arguments (1 for 0)
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/test_executable.rb:145:in `testable?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/interactive_bottleneck.rb:19:in `calc_test_maturity!'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:31:in `block in test_calc_test_maturity'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:29:in `map'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:29:in `test_calc_test_maturity'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
    /usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'


  2) Error:
NewTestClass#test_dirty_test_maturities:
ArgumentError: wrong number of arguments (1 for 0)
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/test_executable.rb:145:in `testable?'
    /home/greg/Desktop/src/Open-Table-Explorer/app/models/interactive_bottleneck.rb:19:in `calc_test_maturity!'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:109:in `block in test_dirty_test_maturities'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:106:in `map'
    /home/greg/Desktop/src/Open-Table-Explorer/test/unit/interactive_bottleneck_test.rb:106:in `test_dirty_test_maturities'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:108:in `block (3 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:205:in `capture_exceptions'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:105:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:256:in `time_it'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:104:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:276:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest/test.rb:103:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:778:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:305:in `run_one_method'
    /usr/lib/ruby/vendor_ruby/minitest.rb:293:in `block (2 levels) in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:292:in `each'
    /usr/lib/ruby/vendor_ruby/minitest.rb:292:in `block in run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:331:in `on_signal'
    /usr/lib/ruby/vendor_ruby/minitest.rb:318:in `with_info_handler'
    /usr/lib/ruby/vendor_ruby/minitest.rb:291:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `block in __run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `map'
    /usr/lib/ruby/vendor_ruby/minitest.rb:152:in `__run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:129:in `run'
    /usr/lib/ruby/vendor_ruby/minitest.rb:56:in `block in autorun'

30 runs, 204 assertions, 0 failures, 2 errors, 0 skips

